go/vt/vttablet/tabletserver/stateful_connection.go

```diff
@@ -51,6 +52,8 @@ type StatefulConnection struct {
 	reservedProps  *Properties
 	tainted        bool
 	enforceTimeout bool
+	timeout        time.Duration
+	timeUsed       *atomic.Value
```

Comment:
any reason for adding atomic here? One connection will be accessed only by one thread at a time, this is protected by a higher order GetAndLock method.

Reply:
It's because of the transaction killer uses this value and which I think can access connections in use in other threads in order to force close ones that are idle or taking too long.

Reply:
If a connection is inUse already we cannot foreclose and only a put can update this value. I believe we do not need to safe guard it. earlier as well, this was not protected by mutex.

Reply:
OK thanks. I had some statements in the wrong order. Fixing that let me get rid of the atomic value.